docs/_data/menu-streaming.yaml

```diff
@@ -0,0 +1,42 @@
+- text: Overview 
```

Comment:
Please add the ASF licence header here

Reply:
Hmm... none of the other files under `_data` have it. Do we want it just for the new files, or should I add it to all the existing _data files?

Reply:
we can fix the new files first to minimize the LOC